README.md

```diff
@@ -4,6 +4,7 @@
 
 > A Leaderboard list app, using webpack and ES6 features, notably modules.
 
+![screenshot](./app_scr![Captura de Pantalla 2021-10-07 a la(s) 8 46 10](https://user-images.githubusercontent.com/68709712/136419795-906c386a-b2eb-4e99-bf38-2b929690dbe6.png)
 
 ### Setup and Install
 
```

Comment:
- [ ] It seems that your application is talking to an API, but you did not provide instructions on how to create a new game id 😄 , because for now, anyone cloning the repo will be using the same game ID which s not really a good idea, So can you please provide instructions on how to generate a new ID for a new game as mentioned in the project requirement below 👇 
![image](https://user-images.githubusercontent.com/55465133/136432541-18f789d4-aba6-4aec-91ea-6c9aa381b5f4.png)